### DISCUSSION TITLE
Can I use florets with Span Catgegorizer

### DISCUSSION THREAD

**Discussion by mbrunecky** [explosion/spaCy#12273] [Help: Coding & Implementations]
I am finding that while switching my NER pipelines from en_core_web_lg to florets does not change any training results, the training uses much less memory - in fact sometimes it may fit into my limited memory GPU.

However, I am trying to use Span Categorizer to be able to use inference confidences, and when I try Span Categorizer with florets, during initialization I get:
`ValueError: [E858] The floret vector table does not support this operation. Use Vectors[key] instead.`

I m using Spacy V3.4, and I checked my configuration using spacy init config. Configuration works just fine using vectors=en_core_web_lg.

Is it a known floret limitation or am I doing something wrong?

**Comment by adrianeboyd** [comment]
Hmm, this is supposed to work and I didn't have trouble running simple examples on my end. Is there anything else custom about your config or additional pipeline components or maybe even your spacy installation? Something related to a custom spancat suggester maybe?

This ran with no issues for the data from the demo project `pipelines/spancat_demo`:

```none
spacy init config -p spancat -o accuracy spancat.cfg
spacy train spancat.cfg --paths.train train.spacy --paths.dev dev.spacy --paths.vectors en_vectors_floret_md
```

Could you provide the full traceback that leads to this error?
